content/en/docs/concepts/crp.md

```diff
@@ -485,6 +485,20 @@ Events:
   Normal  PlacementApplied              3m46s  cluster-resource-placement-controller  Resources have been applied to the selected cluster(s)
   Normal  PlacementRolloutCompleted     3m46s  cluster-resource-placement-controller  Resources are available in the selected clusters
 ```
+
+## ClusterResourcePlacementStatus
+
+The `ClusterResourcePlacementStatus` (CRPS) is a **namespaced resource** that mirrors the PlacementStatus of a corresponding **cluster-scoped** `ClusterResourcePlacement` (CRP) object. It provides namespace-scoped access to cluster-scoped placement status information.
+
+*   **Namespace-scoped access**: Allows users with namespace-level permissions to view placement status without requiring cluster-scoped access
+*   **Status mirroring**: Contains the same placement status information as the parent CRP, but accessible within a specific namespace
+*   **Optional feature**: Only created when `StatusReportingScope` is set to `NamespaceAccessible`, once set `StatusReportingScope` is immutable
+
+
+For a `ClusterResourcePlacement` with `StatusReportingScope` set to `NamespaceAccessible`, only one Namespace resource selector is allowed and it is immutable. Once specified during creation, the namespace resource selector cannot be updated.
```

Comment:
```suggestion
For a `ClusterResourcePlacement` with `StatusReportingScope` set to `NamespaceAccessible`, only **one** Namespace resource selector is allowed and it is immutable. Once specified during creation, the namespace resource selector cannot be updated.
```

Reply:
The namespace resource selector cannot be updated?  What happens when one of the cluster selected left the fleet and we move the namespace to another cluster?